src/t8_geometry/t8_geometry_helpers.c

```diff
@@ -372,6 +372,75 @@ t8_geom_get_triangle_scaling_factor (int edge_index, const double *tree_vertices
   return scaling_factor;
 }
 
+double
+t8_geom_get_scaling_factor_of_edge_on_face_prism (const int edge, const int face, const double *ref_coords)
```

Comment:
Why can't we use the triangle function for that?

Reply:
The triangle function does not work here, because it follows a different approach (computation of intersection).
I would rather adapt the triangle function to this kind of function, because it is also used for tetrahedral elements and we are therefore more consistent.